README.md

```diff
@@ -92,4 +92,40 @@ server.stop()
 ```
 
 ## Utils
-...
+### Logging
+You can configurate logs as you want by config.
+
+Example:
+
+```py
+from sc_kpm.constants import MAIN_LOGGER
```

Comment:
change name from MAIN_LOGGER to LOGGER_NAME, we have no another loggers. Also I want to import LOGGER_NAME directly from the sc_kpm.

Reply:
actually, you can import it directly from sc_kpm

Reply:
no, I think better to use full import. it can brake import order